### PR TITLE
docs: minor changes to the `merge-reports` documentation

### PR DESCRIPTION
**Pull request by aslushnikov**

### CODE REVIEW

File: docs/src/test-sharding-js.md [hunk 1]
```diff
@@ -45,7 +47,14 @@ This will produce a standard HTML report into `playwright-report` directory.
 
 One of the easiest ways to shard Playwright tests across multiple machines is by using GitHub Actions matrix strategy. For example, you can configure a job to run your tests on four machines in parallel like this:
 
-```yaml
+```yaml title=".github/workflows/playwright_tests.yml"
```
Comment (mxschmitt) [inline docs/src/test-sharding-js.md:50]:
```suggestion
```yaml title=".github/workflows/playwright.yml"
```

We call it `playwright.yml`: https://github.com/microsoft/create-playwright/blob/c1aae203f3905d7f734a3efb7d1c4f517ff3e36b/src/generator.ts#L175

File: docs/src/test-sharding-js.md [hunk 2]
```diff
@@ -62,20 +71,20 @@ jobs:
...
-        name: blob-report-${{ github.run_attempt }}
+        name: all-blob-reports--run-${{ github.run_attempt }}
         path: blob-report
         retention-days: 1
 ```
 
 After all shards have completed, run a separate job that will merge the reports and produce a combined HTML report.
 
-```yaml
+```yaml title=".github/workflows/playwright_tests.yml"
```
Comment (mxschmitt) [inline docs/src/test-sharding-js.md:87]:
```suggestion
```yaml title=".github/workflows/playwright.yml"
```

File: docs/src/test-sharding-js.md [hunk 3]
```diff
@@ -143,13 +153,13 @@ Supported options:
 
   Which report to produce. Can be multiple reporters separated by comma.
 
-  Example: `npx playwright merge-reports ./blob-reports --reporter=html,github`
+  Example: `npx playwright merge-reports --reporter=html,github`./blob-reports
```
Comment (mxschmitt) [inline docs/src/test-sharding-js.md:156]:
```suggestion
  Example: `npx playwright merge-reports --reporter=html,github ./blob-reports`
```

File: docs/src/test-sharding-js.md [hunk 4]
```diff
@@ -93,17 +102,18 @@ jobs:
...
 
     - name: Merge into HTML Report
-      run: npx playwright merge-reports ./all-blob-reports --reporter html
+      run: npx playwright merge-reports --reporter html ./all-blob-reports 
 
     - name: Upload HTML report
       uses: actions/upload-artifact@v3
       with:
-        name: html-report-${{ github.run_attempt }}
+        name: html-report--run-${{ github.run_attempt }}
```
Comment (mxschmitt) [inline docs/src/test-sharding-js.md:114]:
```suggestion
        name: html-report-run-${{ github.run_attempt }}
```

File: docs/src/test-sharding-js.md [hunk 5]
```diff
@@ -62,20 +71,20 @@ jobs:
...
     - name: Run Playwright tests
-      run: npx playwright test --shard ${{ matrix.shard }}/4
+      run: npx playwright test --shard ${{ matrix.shard }}/4 --reporter blob
 
     - name: Upload blob report to GitHub Actions Artifacts
       if: always()
       uses: actions/upload-artifact@v3
       with:
-        name: blob-report-${{ github.run_attempt }}
+        name: all-blob-reports--run-${{ github.run_attempt }}
```
Comment (mxschmitt) [inline docs/src/test-sharding-js.md:80]:
```suggestion
        name: all-blob-reports-run-${{ github.run_attempt }}
```

File: docs/src/test-sharding-js.md [hunk 6]
```diff
@@ -62,20 +71,20 @@ jobs:
       run: npx playwright install
 
     - name: Run Playwright tests
-      run: npx playwright test --shard ${{ matrix.shard }}/4
+      run: npx playwright test --shard ${{ matrix.shard }}/4 --reporter blob
```
Comment (yury-s) [inline docs/src/test-sharding-js.md:74]: We mentioned above that one has to add 'blob' reporter to the config. So I'd rather drop this parameter here as it is not what we ourselves do.

File: docs/src/test-sharding-js.md (hunk 5 above)
Comment (yury-s) [inline docs/src/test-sharding-js.md:80]: all-blob-reports--run -> all-blob-reports--attempt?

**Review by yury-s — APPROVED** [review]

**Pull request merged**